crowdin.yml

```
@@ -17,6 +17,12 @@ files:
   - source: /jekyll/_includes/snippets/*.adoc
     translation: /jekyll/_includes/snippets/%two_letters_code%/%original_file_name%
     update_option: update_as_unapproved
+  - source: /jekyll/_includes/snippets/faq/*.adoc
+    translation: /jekyll/_includes/snippets/%two_letters_code%/%original_file_name%
```

I would expect this to be either:
```suggestion
    translation: /jekyll/_includes/snippets/%two_letters_code%/faq/%original_file_name%
```

OR 

```suggestion
    translation: /jekyll/_includes/snippets/faq/%two_letters_code%/%original_file_name%
```

Which one is correct will depend on how the code that consumes these strings is expecting these to be located; I am not sure that is set up yet, since either of the suggested directory formats does not yet exist.

---

Oh of course, thanks for this!